app/javascript/components/enrollment/steps/Contact.js

```diff
@@ -310,9 +288,17 @@ class Contact extends React.Component {
                     {this.state.errors['secondary_telephone']}
                   </Form.Control.Feedback>
                 </Form.Group>
+                {this.state.current.patient?.preferred_contact_method?.includes('SMS') && this.state.current.blocked_sms && (
```

Comment:
do we need a warning for the 'Telephone Call' option too? or does the blocking only apply to text messages?

Reply:
I'm pretty sure it's just for texts.  I didn't change any of that code, @mayerm94 might know?

Reply:
This blocking only applies to SMS